app/src/main/res/layout/main.xml

```diff
@@ -5,14 +5,81 @@
     android:layout_height="match_parent" >
 
     <!-- The main content view -->
-
-    <FrameLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:id="@+id/list_fragment_container"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:padding="5dp">
+
+        <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+            android:id="@+id/list_fragment_container"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_above="@+id/stream_actv"
+            android:layout_alignParentLeft="true"
+            android:layout_alignParentStart="true" />
+
+        <AutoCompleteTextView
+            android:id="@+id/stream_actv"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_above="@+id/message_et"
+            android:layout_alignParentLeft="true"
+            android:layout_alignParentStart="true"
+            android:dropDownWidth="fill_parent"
+            android:hint="Stream" />
+
+
+        <TextView
+            android:id="@+id/textView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_above="@+id/message_et"
+            android:layout_toRightOf="@+id/stream_actv"
+            android:padding="6dp"
+            android:text=">"
+            android:textAppearance="?android:attr/textAppearanceLarge" />
+
+        <AutoCompleteTextView
+            android:id="@+id/topic_actv"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_above="@+id/message_et"
+            android:layout_toRightOf="@+id/textView"
+            android:dropDownWidth="fill_parent"
+            android:maxLines="1"
+            android:hint="Topic" />
+
+        <EditText
+            android:id="@+id/message_et"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true"
+            android:layout_toLeftOf="@+id/send_btn"
+            android:layout_toStartOf="@+id/send_btn"
+            android:hint="Tap on a message to send reply"
+            android:inputType="textCapSentences|textMultiLine"
+            android:maxLines="4" />
+
+
+        <ImageView
+            android:id="@+id/send_btn"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignEnd="@+id/list_fragment_container"
+            android:layout_alignRight="@+id/list_fragment_container"
+            android:layout_alignTop="@+id/message_et"
+            android:src="@drawable/ic_send_black_24dp"
```

Comment:
it's best practice to introduce the drawable in the same commit that you use it in, or at least before the commit that you use the drawable in.  that makes it easier to roll back the change if need be.